bigquery/deploy/README.md

```diff
@@ -69,3 +72,10 @@
     kubectl exec -it ES_CLIENT_POD -- /bin/bash
     curl EXTERNAL_IP:9200/_cat/indices?v
     ```
+
+* Delete cluster
```

Comment:
Instructions for deleting a cluster aren't in the scope of setting up an Elasticsearch instance, could we remove this?

Reply:
Moved to a different section